INSTALL-MACOS.md

```
@@ -0,0 +1,203 @@
+./OliveTin
+```
+
+Or, if you installed it to `/usr/local/bin` and keep your config elsewhere:
+
+```sh
+OliveTin -configdir /usr/local/etc/OliveTin
+```
+
+Then open the web interface at:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)